File: C/arrays/linear_search.c
#include <stdio.h>
int main(){
    int n,key; scanf("%d",&n);
    int arr[n]; for(int i=0;i<n;i++) scanf("%d",&arr[i]);
    printf("Enter key: "); scanf("%d",&key);
    int found=-1;
    for(int i=0;i<n;i++) if(arr[i]==key){ found=i; break; }
    if(found!=-1) printf("Found at index: %d",found);
    else printf("Not found");
}

File: python/arrays/linear_search.py
# Linear Search in Python
n = int(input("Enter size of array: "))
arr = list(map(int, input(f"Enter {n} elements: ").split()))
key = int(input("Enter element to search: "))

index = -1
for i in range(n):
    if arr[i] == key:
        index = i
        break

if index != -1:
    print("Element found at index:", index)
else:
    print("Element not found in the array.")
